packages/core/src/extensions/Blocks/nodes/Block.module.css

```diff
@@ -3,21 +3,11 @@ BASIC STYLES
 */
 
 .blockOuter {
-  /* padding-bottom: 15px; */
-  font-size: 16px;
   line-height: 1.5;
-  transition: all 0.2s;
-  font-weight: normal;
-}
-
-.block {
```

Comment:
I didn't see this moved to a different file, was the transition just redundant?

Reply:
yes I think so, but maybe you can double check if all animations still work